openapi/kas-fleet-manager-private.yaml

```diff
@@ -19,8 +19,56 @@ servers:
 tags:
   - name: Agent Clusters
     description: ""
+  - name: observatorium-proxy
+    description: Observatorium proxy related endpoints
 
 paths:
+  # observatorium proxy related endpoints
+  '/api/kafkas_mgmt/v1/obs-proxy-verification/{cluster_external_id}':
```

Comment:
Can we add a description to the endpoint?

Reply:
I'll address this and the url format comment in a new PR